doc/source/tune/api_docs/integration.rst

```diff
@@ -68,7 +68,7 @@ Weights and Biases (tune.integration.wandb)
 .. autoclass:: ray.air.callbacks.wandb.WandbLoggerCallback
     :noindex:
 
-.. autofunction:: ray.tune.integration.wandb.wandb_mixin
+.. autofunction:: ray.air.integrations.wandb.setup_wandb
```

Comment:
Is there a better option for the directory structure?

Currently, `WandbLoggerCallback` and `setup_wandb` are in two different packages. I think we can just have `setup_wandb` also in `ray.air.callbacks.wandb`.

We can also rename `ray.air.callbacks` to `ray.air.integrations`.

Reply:
`callbacks` is very specific so I'd rather not have `setup_wandb` in it. It can also become a problem in the future when we may have other non-callback integrations.

I'm happy to move the `ray.air.callbacks` to `integrations`, I'll update the PR soon to reflect this